src/runtime/virtcontainers/qemu.go

```diff
@@ -734,7 +734,7 @@ func (q *qemu) CreateVM(ctx context.Context, id string, network Network, hypervi
 		return err
 	}
 
-	devices, ioThread, kernel, err := q.buildDevices(ctx, kernelPath)
+	devices, SCSIioThread, kernel, err := q.buildDevices(ctx, kernelPath)
```

Comment:
IMO, if only virtio-scsi device will use `ioThread`, just comment about it is enough. There's no point here change the name of the `ioThread` to `SCSIioThread`.

Reply:
OK, I will add comments in the next version.